src/components/viewsComponents/dashboard/elements/announcements/Announcements.tsx

```diff
@@ -0,0 +1,36 @@
+import EventIcon from '@mui/icons-material/Event';
+import { Typography, List, ListItem, Paper, Box } from '@mui/material';
+import { format, parseISO } from 'date-fns';
+import { announcementsData } from '../../constants/AnnouncementsData';
+
+const Announcements = () => {
+  const announcements = announcementsData;
+
+  return (
+    <Paper className="PaperContainer">
+      <Typography variant="h5" className="Title" gutterBottom>
+        Latest Messages from Administration
+      </Typography>
+      <List>
+        {announcements.map(({ id, title, body, date }) => (
+          <ListItem key={id} className="ListItem" sx={{ gap: 1 }}>
+            <Typography variant="subtitle1" component="div" className="Title">
```

Comment:
Why typographies as component div ?

Reply:
It was preventing DOM structure error in previous version. It is now unnecessary.